applications/GeoMechanicsApplication/custom_constitutive/coulomb_with_tension_cut_off_impl.h

```diff
@@ -0,0 +1,44 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohamed Nabi
+//                   Wijtze Pieter Kikstra
+//                   Anne van de Graaf
+//
+
+#pragma once
+
+#include "coulomb_yield_surface.h"
+#include "tension_cutoff.h"
+
+namespace Kratos
+{
+
+class Properties;
```

Comment:
Let's also explicitly forward declare `class Serializer;` here. We now get it indirectly through one of the `#include`s, but I feel it's better to be explicit about it.
```suggestion
class Properties;
class Serializer;
```